lib/pokerwars/game.ex

```diff
@@ -45,6 +51,19 @@ defmodule Pokerwars.Game do
     |> deal_two_cards_to_each_player
   end
 
+  defp take_blinds(game) do
+    players = game.players
+
+    [ first_player | [ second_player | other_players ] ] = players
+
+    first_player = %{first_player | stack: first_player.stack - 10}
+    second_player = %{second_player | stack: second_player.stack - 20}
```

Comment:
Would be nice to have a value inside the game called `small_blind`, because big blind is always double the small blind.